docs/5.x/pack.md

```diff
@@ -525,7 +525,8 @@ systemOptions:
 
   # Docker section allows to customize docker
   docker:
-    # Storage backend used, supported: "devicemapper" (default), "overlay", "overlay2"
+    # Storage backend used, supported: "devicemapper", "overlay", "overlay2" (default)
+    # Note: devicemapper is no longer available from gravity 5.3.4 or later.
```

Comment:
I would then just remove the reference to it from here and above. IIRC we have a note about it on the System Requirements page.